<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="url_box">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>11</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="url_text">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>URL: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_playlist_check">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Playlist Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="custom_dir_name_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Custom Dir Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="custom_dir_name_box">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="curr_download_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Download:</string>
    </property>
   </widget>
   <widget class="QLabel" name="curr_download_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download Progress:</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 %</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Warning List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="download_settings_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download Settings:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="download_setting_option">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>144p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>240p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>360p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>480p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>720p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1080p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>audio</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
